signac/contrib/project.py

```diff
@@ -65,18 +67,37 @@ class JobSearchIndex(object):
 
     The JobSearchIndex allows to search for job_ids,
     that are part of an index, which match specific
-    statepoint filters or job document filters.
+    state point filters or job document filters.
 
-    :param index: A document index.
-    """
+    Parameters
+    ----------
+    index :
+        A document index.
+    _trust : bool
+        Whether to skip document validation on insertion into the internal
+        Collection (Default value = False).
 
+    """
     def __init__(self, index, _trust=False):
         self._collection = Collection(index, _trust=_trust)
 
     def __len__(self):
         return len(self._collection)
 
     def _resolve_statepoint_filter(self, q):
+        """Resolve state point based on filter.
+
+        Parameters
+        ----------
+        q : dict
+            Filter used for state point selection.
+
+        Yield
```

Comment:
Unfortunately I think this has to be plural
```suggestion
        Yields
```